starters/gatsby-wordpress-starter/lib/privatePosts.js

```diff
@@ -0,0 +1,31 @@
+const { gql, GraphqlClientFactory } = require('@pantheon-systems/wordpress-kit')
+
+const client = new GraphqlClientFactory(`${process.env.WPGRAPHQL_URL}`).create()
```

Comment:
Not sure there is any need to wrap this in backticks.
```suggestion
const client = new GraphqlClientFactory(process.env.WPGRAPHQL_URL).create()
```